rllib/algorithms/appo/appo_tf_policy.py

```diff
@@ -81,10 +81,13 @@ def __init__(
             # First thing first, enable eager execution if necessary.
             base.enable_eager_execution_if_necessary()
 
+            old_stack = not config.get("_enable_learner_api", False)
```

Comment:
if we are on the new-stack, we don't need any mixins that only affect the loss, optimizer, grad computations, etc..
However, since I really don't want to create yet another sub-class or new mixin schema, we'll solve this here by simply not calling the constructors of certain mixins.